book/_toc.yml

```diff
@@ -0,0 +1,15 @@
+# Table of contents
+# Learn more at https://jupyterbook.org/customize/toc.html
+
+format: jb-book
+root: intro
+parts:
+- caption: Details
+  chapters:
+    - title: AGU24 Homepage
+      url: https://www.agu.org/annual-meeting
```

Comment:
At the moment this link seems to resolve the website for the 2024 fall meeting. But I think later we have to update the link, as in 2025 this link will probably resolve the 2025 fall meeting website >.<.

Reply:
Yeah, I wished they just used a permanent URL like https://www.agu.org/fall-meeting-2024 directly. Will try to remember to change this next year!